src/components/map/UserCard.tsx

```diff
@@ -21,6 +21,8 @@ const UserCard = ({ user, onClose }: UserCardProps) => {
   const [lastActive, setLastActive] = useState(user.lastActive); // TODO: setLastActive when user is active?
```

Comment:
what if this is undefined?

Reply:
instead of addressing `UserCard` in this PR we should open a new one for that, since quite a lot of components are using it. it is also functionally different from this create group PR.